src/draw_request.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <memory>
```

Comment:
I was trying to use smart pointers before, but I realize now it's not necessary. I'll delete this line.